docker/localsettings.py

```diff
@@ -52,6 +59,9 @@
             'TEST': {
                 'SERIALIZE': False,
             },
+            'PLPROXY': {
+                'SHARDS': [1, 2],
```

Comment:
the old config had `'p2': [2, 3]`. Was the change from `[2, 3]` to `[1, 2]` intentional? The example in the PR description has the same issue.

Reply:
No it wasn't intentional